specification/security/resource-manager/Microsoft.Security/preview/2020-08-06-preview/insights.json

```diff
@@ -0,0 +1,254 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Security Center",
+    "description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
+    "version": "2020-08-06-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/{resourceId}/providers/Microsoft.Security/Insights": {
+      "get": {
+        "x-ms-examples": {
+          "List insights of an Azure storage resource": {
+            "$ref": "./examples/Insights/ListStorageInsights_example.json"
+          }
+        },
+        "tags": [
+          "Insights"
+        ],
+        "description": "Insights list of a specific resource",
+        "operationId": "Insights_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceId"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/InsightsList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/{resourceId}/providers/Microsoft.Security/Insights/{insightName}": {
```

Comment:
Do we want "Insights" (upper I) or "insights"?

You used `Microsoft.Security/insights` for the type name. I think we should go with lowercase here, no?

Reply:
Right, will update.